package.json

```diff
@@ -6,7 +6,7 @@
   "gypfile": true,
   "scripts": {
     "test-timeouts": "tape test/timeouts.js",
-    "test": "standard && tape test/net.js test/sockets.js test/udp.js",
```

Comment:
The test system isn't running the timeouts test which seemed like an oversight, by moving to `*.js` all tests are run to prevent that.

Reply:
Timeout tests were skipped because they can be really slow so so just test them manually

Reply:
Returned to previous functionality: https://github.com/mafintosh/utp-native/pull/42/commits/cdbb968c4e6d44d78997fa2257cb9da72a010d7b

Reply:
When I test the code I always used `node test/net.js` anyways instead of `npm test` for quicker testing. `npm test` imo. should test the whole thing and also run during PR's/Commits.